### PR TITLE
Create Releases “2022-07-06-june-2022”

### PR DESCRIPTION
**Pull request by amanifarooque**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by amanifarooque** [comment]
Link to release notes: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/releases/2022-07-06-june-2022/about/updates/releases/june-2022.html

**Comment by amanifarooque** [comment]
@dawnpm - I can't add Rob as a reviewer to this PR to review as dev lead representative. Can we adjust the permissions to whatever group makes sense so he can review?

File: _releases/2022-07-06-june-2022.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```
+<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<url>
+<loc>https://example.gov/my-page.html</loc>
+<lastmod>2022-07-06T00:00:00+00:00</lastmod>
+</url>
+</urlset>
+``` 
+* **Security Enhancements**: We've improved our ability to respond to external attacks to the system.
+* **ElasticSearch Upgrade**: We’ve continued progress on upgrading our ElasticSearch version across our system.
```
Comment (MyNameIsMissing) [inline _releases/2022-07-06-june-2022.md:30]: The apostrophe here and the one above don't match.  Not sure why.  I recommend making them consistent.  There is a small potential for some future system to choke on the difference.  The second one is probably from copy-paste from MS Word I'm guessing.

**Review by MyNameIsMissing — CHANGES_REQUESTED** [review]
Small item I would change if not terribly inconvenient to do so.

**Review by MyNameIsMissing — DISMISSED** [review]
LGTM

**Comment by dawnpm** [comment]
@amanifarooque Can we do a block indent of the sitemap examples, so that they align with the bullet point? I was confused by the alignment.

**Pull request merged**